databrowser/src/components/download/DownloadMenu.vue

```diff
@@ -67,22 +67,34 @@ SPDX-License-Identifier: AGPL-3.0-or-later
                 <!-- Status icons -->
                 <IconCheck
                   v-if="download.status === 'completed'"
-                  class="size-7 shrink-0 rounded-full bg-green-500 text-white"
+                  class="size-5 shrink-0 rounded-full bg-green-500 text-white"
                 />
                 <IconErrorWarning
                   v-else-if="download.status === 'failed'"
-                  class="size-7 shrink-0 rounded-full bg-hint-error text-white"
+                  class="size-5 shrink-0 rounded-full bg-hint-error text-white"
                 />
                 <InfiniteSpinner
                   v-else-if="download.status === 'in-progress'"
-                  class="size-7 shrink-0"
+                  class="size-5 shrink-0"
                 />
 
-                <!-- Download name -->
-                <div class="flex grow flex-col overflow-auto">
-                  <span>{{ download.name }}</span>
-                  <span v-if="download.error" class="text-hint-error">
-                    {{ download.error }}
+                <div class="flex grow items-center overflow-hidden">
+                  <!-- Download name -->
+                  <span
+                      class="truncate"
+                      :class="download.error ? 'w-1/2' : 'w-full'"
+                      :title="download.name"
+                  >
+                    {{ download.name }}
+                  </span>
+                  <!-- Download error -->
+                  <span
+                      v-if="download.error"
+                      class="ml-2 w-1/2 shrink-0 truncate text-hint-error cursor-copy"
+                      :title="download.error"
```

Comment:
The `title` attribute is set on this `span`, in addition there is the `SimpleTooltip` component below, and both show the same content. As result, when you hover over an error, you'll see two tooltips. Which one is shown depends if you move the mouse over the error message or the retry button. I'd opt for one solution (see also other comment in this file regarding tooltips)